### PR TITLE
Default selectId function for EntityAdapter

### PR DESCRIPTION
**Pull request by PhillipZada**
See #378 

**What is the current behavior?**

I have several entities, and I use always id property for the primary key. So, to create the EntityAdater I must repeat the code like:

```
export const myEntityAdapter: EntityAdapter<MyEntity> = createEntityAdapter<MyEntity>({
  selectId: (myEntity: MyEntity) => myEntity.id
});
```

**Expected behavior:**

This could be greate if createEntityAdapter uses a default selectId method which will take id property of object as a primary key. So I just need call this function

```
export const myEntityAdapter: EntityAdapter<MyEntity> = createEntityAdapter<MyEntity>();
```

### CODE REVIEW

File: modules/entity/src/create_adapter.ts [hunk 1]
```diff
@@ -10,10 +10,12 @@ import { createSelectorsFactory } from './state_selectors';
 import { createSortedStateAdapter } from './sorted_state_adapter';
 import { createUnsortedStateAdapter } from './unsorted_state_adapter';
 
-export function createEntityAdapter<T>(options: {
+export function createEntityAdapter<T>(options?: {
```
Comment (brandonroberts) [inline modules/entity/src/create_adapter.ts]: Instead of making `options` optional, make `selectId` optional and provide a default below

File: modules/entity/src/create_adapter.ts [hunk 2]
```diff
@@ -10,10 +10,12 @@ import { createSelectorsFactory } from './state_selectors';
...
   selectId: IdSelector<T>;
   sortComparer?: false | Comparer<T>;
 }): EntityAdapter<T> {
+  options = options || { selectId: (instance: any) => instance.id };
+
   const { selectId, sortComparer }: EntityDefinition<T> = {
     sortComparer: false,
```
Comment (brandonroberts) [inline modules/entity/src/create_adapter.ts]: Added a default `selectId` function here under the `sortComparer`

File: modules/store/src/store_module.ts [hunk 3]
```diff
@@ -149,12 +149,16 @@ export class StoreModule {
   ): ModuleWithProviders;
```
Comment (brandonroberts) [inline modules/store/src/store_module.ts]: Do these changes in a separate commit

Comment (PhillipZada) [reply modules/store/src/store_module.ts]: Hey Brandon, these changes are happening automatically as per the precommit script. Any suggestions on how to stop it.

File: modules/entity/src/create_adapter.ts (hunk 2 above)
Comment (PhillipZada) [reply modules/entity/src/create_adapter.ts]: Done

**Comment by PhillipZada** [comment]
Hey @brandonroberts,

The original request was to make it possible to create an adaptor using this 

```
export const myEntityAdapter: EntityAdapter<MyEntity> = createEntityAdapter<MyEntity>();
```

I removed the optional options flag and replaced it with a default value if not provided. Hope this meets requirements.

File: modules/store/src/store_module.ts (hunk 3 above)
Comment (brandonroberts) [reply modules/store/src/store_module.ts]: Don't worry about it then. Must have merged another change that didn't get formatted.

**Comment by brandonroberts** [comment]
Thanks!
